examples/wesleycsj-helloWorld.cpp

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
Why the filename in camelcase too? Any reason in particular to choose `examples/wesleycsj-helloWorld.cpp `?. kabeb-case it seems a more consistent fit with the other examples: `examples/wesleycsj-hello-world.cpp`. 

- lerax-square-platform.cpp

Reply:
Fixed now. There wasn't no conversation about it before.